client/devdocs/design/gutenberg-blocks.jsx

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
If we add  `/** @format */` above this line, prettier, will handle whitespace formatting for us.